docker/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM openjdk:11.0.5-jre-slim-buster
+
+COPY orion /opt/orion/
+WORKDIR /opt/orion
+
+# Install latest libsodium
```

Comment:
nit: we aren't installing the latest, we are installing a specific version (which is a good thing). Just change the comment to `Install Libsodium` :)

Reply:
done